### PR TITLE
Create 20191012-Niu-Wa-Yu-Tou

### PR DESCRIPTION
**Pull request by Yingzhang1122**
本期比较轻松，请大家检查指正！
#280

### CODE REVIEW

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+# 准备大蒜40克切成小粒备用
+Cut 40 grams of garlic into small pieces
+
+0:01:13.900,0:01:16.320
+# 准备适量的红泡椒切成小粒备用
+Cut a moderate amount of red fermented chili into small pieces
+
+0:01:19.880,0:01:22.380
+# 准备麻辣火锅底料50克切散备用
+Cut up 50 grams of spicy hot-pot ingredient
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: Cut up 50 grams of spicy hot-pot soup base

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+
+0:01:22.380,0:01:24.300
+# 底料不受任何品牌限制
+You can use the ingredient of any brand
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: You can use any brand you like

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+# 准备豆芽1小把备用
+Prepare a handful of bean sprouts
+
+0:01:44.040,0:01:46.380
+# 准备芹菜几根切成小段放入盆中垫底
+Cut some celery into pieces and put it in the bowl
+
+0:01:46.440,0:01:48.900
+# 同学们也可以根据个人喜好添加底菜
+You can use your favorite vegetables as the base vegetable #请检查这里的表达
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: 没问题

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+# 准备干辣椒一小把剪成小节备用
+Cut a handful of dried red chili into small pieces with scissors
+
+0:01:38.600,0:01:40.800
+# 碗中再加入适量的青花椒备用
+Add some Sichuan peppercorn into the bowl
+
+0:01:40.800,0:01:42.200
+# 下一步开始准备打底菜
+Then, prepare the "Da di cai" (vegetables at the bottom of the bowl) #请检查这里的表达
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: 没问题

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+# 然后我们再次把锅烧热
+Then, heat up the wok again
+
+0:02:21.700,0:02:24.840
+# 锅烧热之后加入熟菜籽油50克烧至6成热
+Add 50 grams of "Shu cai zi you" (rapeseed oil that has been heated) and heat it until 60% hot
+
+0:02:27.860,0:02:30.820
+# 油温6成热之后加入准备好的料头下锅爆香
+Add the prepared "liao tou" (seasoning ingredients, such as garlic, ginger, etc) and cook them until aromatic #请检查这里的表达
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: Add the prepared "liao tou" (aromatics, such as garlic, ginger, etc) and cook them until fragrant

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+
+0:02:35.885,0:02:38.465
+# 中途再加入切好的火锅底料和酸菜
+Add the chopped hot-pot ingredient and fermented vegetable into the wok halfway
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: Add the chopped hot-pot base and fermented vegetable into the wok halfway

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+
+0:02:45.285,0:02:48.385
+# 然后转小火翻炒3分钟将底料炒香炒化
+Cook with low heat for 3 minutes to melt the hot-pot ingredient
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: ook with low heat for 3 minutes to melt the hot-pot base

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+# 牛蛙鱼头
+Niu Wa Yu Tou (Bullfrog with fish head)
+
+0:00:05.100,0:00:06.940
+# 首先我们准备牛蛙4只
+Firstly, prepare 4 bullfrogs
+
+0:00:06.940,0:00:09.400
+# 去掉外皮和内脏清洗干净备用
+Remove the skin and internal organs and wash it thoroughly
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]:
```suggestion
Remove the skin and internal organs and wash them thoroughly
```

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 9]
```diff
@@ -0,0 +1,277 @@
...
+# 锅中加入食用盐1大勺（约5克）
+Add a big spoon of salt (~ 5 grams)
+
+0:03:08.120,0:03:09.620
+# 加入胡椒粉1勺（约1克）
+A spoon of pepper powder (~ 1 gram)
+
+0:03:10.245,0:03:11.685
+# 加入五香粉1勺（约0.5克）
+A spoon of "Wu xiang fen" (~ 0.5 gram)
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: Five spice powder

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 10]
```diff
@@ -0,0 +1,277 @@
...
+# 考虑宰杀这一步比较血腥
+Since the butchering step is relatively bloody
+
+0:00:11.540,0:00:14.535
+# 所以嫌麻烦的同学可以交给摊主处理
+Please leave it to the shopkeeper if you don't want to do it
+
+0:00:14.535,0:00:17.085
+# 中途用剪刀去掉牛蛙的脚趾
+Remove the fingers of the frog with scissors
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]:
```suggestion
Remove the fingers of the frogs with scissors
```

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 11]
```diff
@@ -0,0 +1,277 @@
...
+# 加入胡椒粉1勺（约1克）
+A spoon of pepper powder (~ 1 gram)
+
+0:03:10.245,0:03:11.685
+# 加入五香粉1勺（约0.5克）
+A spoon of "Wu xiang fen" (~ 0.5 gram)
+
+0:03:12.475,0:03:13.925
+# 加入鸡精2勺（约4克）
+Two spoons of essence of chicken  (~ 4 grams)
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: Two spoons of chicken bouillon powder

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 12]
```diff
@@ -0,0 +1,277 @@
...
+# 中途用剪刀去掉牛蛙的脚趾
+Remove the fingers of the frog with scissors
+
+0:00:19.625,0:00:21.895
+# 然后再从中间剪开备用
+Then cut it open in the middle for further use
+
+0:00:25.915,0:00:28.945
+# 牛蛙处理好之后 下一步开始处理鱼头
+When the frog is done, deal with the fish head
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]:
```suggestion
When the frog is done, move on to the fish head
```
或者 processs之前....deal with感覺好像是「處理一個問題/麻煩人」的意思  XD

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 13]
```diff
@@ -0,0 +1,277 @@
...
+# 将油温烧至4成热之后加入准备好的辣椒和花椒
+Heat the oil till 40% hot and add dried chili and Sichuan peppercorns
+
+0:04:27.985,0:04:30.555
+# 然后转大火不停的搅动辣椒
+Then turn to high heat and stir the chili
+
+0:04:30.555,0:04:33.555
+# 将辣椒烧至棕色之后再一次性泼在牛蛙和鱼头上面
+When the chili turns brown, pour the oil onto the frog and fish head at a time
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: When the chili turns brown, pour the oil onto the frog and fish head in one go

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 14]
```diff
@@ -0,0 +1,277 @@
...
+# 然后再从中间剪开备用
+Then cut it open in the middle for further use
+
+0:00:25.915,0:00:28.945
+# 牛蛙处理好之后 下一步开始处理鱼头
+When the frog is done, deal with the fish head
+
+0:00:28.945,0:00:30.480
+# 准备鲜活的花鲢一条
+Prepare a healthy, live silver carp
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]:
```suggestion
Prepare a healthy, living silver carp
```
不過感覺都有點怪....健康...仍然生存的...XD
或許fresh之類？畢竟外國人很少買得到活魚

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 15]
```diff
@@ -0,0 +1,277 @@
...
+# 将鱼敲晕之后去除鱼鳞和内脏
+Stun it and remove the scales and internal organs
+
+0:00:34.500,0:00:36.880
+# 然后将鱼头取下清洗干净备用
+Separate the fish head from the body and wash it thoroughly for further use
+
+0:00:37.835,0:00:40.795
+# 处理的时候可以从鱼头背部下刀切开
+Cut the head open from the backside of the fish
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]:
```suggestion
Cut the head open from the backside of the 竹水日木
```

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 16]
```diff
@@ -0,0 +1,277 @@
...
+# 然后转大火不停的搅动辣椒
+Then turn to high heat and stir the chili
+
+0:04:30.555,0:04:33.555
+# 将辣椒烧至棕色之后再一次性泼在牛蛙和鱼头上面
+When the chili turns brown, pour the oil onto the frog and fish head at a time
+
+0:04:40.435,0:04:42.375
+# 最后再撒上适量的香菜即可
+Finally, sprinkle some coriander on the surface
```
Comment (NoraZheng) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: Finally, garnish with coriander leaves

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 17]
```diff
@@ -0,0 +1,277 @@
...
+# 这样更容易取鳃和清洗
+Such that you can remove the gill and wash it easily
+
+0:00:52.240,0:00:54.700
+# 主料备齐之后下一步开始准备辅料
+When all the major ingredients are ready, start preparing the Fuliao (side ingredients)
+
+0:00:54.700,0:00:57.800
+# 准备酸菜一小把清洗干净之后切成小块备用
+Prepare some fermented vegetables and cut it into small pieces
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: fermented是發酵吧？不過吃的我實在不知道

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 18]
```diff
@@ -0,0 +1,277 @@
...
+
+0:00:57.800,0:00:59.800
+# 加酸菜的目的是增鲜提味
+The purpose of this step is to increase the savory flavor in the dish
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]:
```suggestion
The purpose of this step is to enhance the savory flavor in the dish
```

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv (hunk 1 above)
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]:
```suggestion
Smash 50 grams of spicy hot-pot ingredient
```
他剁得很爛耶....

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 19]
```diff
@@ -0,0 +1,277 @@
...
+# 然后转小火翻炒3分钟将底料炒香炒化
+Cook with low heat for 3 minutes to melt the hot-pot ingredient
+
+0:02:53.040,0:02:55.800
+# 底料炒香之后加入适量的清水开大火烧开
+Then add a moderate amount of water and boil with high heat
+
+0:03:04.140,0:03:05.860
+# 大火烧开之后开始调味
+After boiled, season the soup
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]:
```suggestion
After boiled, put in the seasoning
```
感覺他們的"soup"真是要來吃的....

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv (hunk 9 above)
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: 要說明一下這是甚麼東東嗎？

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 20]
```diff
@@ -0,0 +1,277 @@
...
+# 加入白糖少许中和辣味
+And some sugar to neutralize the spicy flavor
+
+0:03:19.765,0:03:21.725
+# 最后再加入生抽酱油8克
+Finally, add 8 grams of light soy sauce
+
+0:03:24.740,0:03:27.520
+# 调好味之后将准备好的鱼头和牛蛙下锅
+After seasoning, put the frog and the fish head into the wok
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: 蛙已碎了，應該不用frogs了....XD

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 21]
```diff
@@ -0,0 +1,277 @@
...
+# 因为牛蛙和鱼头是现杀所以不需要提前腌制
+You don't need to marinade them in advance because they are fresh
+
+0:03:40.120,0:03:42.620
+# 主料下锅之后转中小火烧8分钟
+Boil them with low heat for 8 minutes
+
+0:03:42.620,0:03:44.820
+# 这一步的目的是将其煮熟煮至入味
+The purpose of this step is to cook them and let the flavor go inside
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]:
```suggestion
The purpose of this step is to thoroughly cook them and let the flavor go inside
```

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv [hunk 22]
```diff
@@ -0,0 +1,277 @@
...
+
+0:03:50.540,0:03:53.040
+# 8分钟之后将主料捞出放入盆中
+8 minutes later, take out the major ingredients and put them into the bowl
+
+0:04:04.520,0:04:06.440
+# 然后再加入适量的原汤
+Then add some original soup
```
Comment (vincenthychow) [inline subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: 就是「湯」和"soup"的微妙不同吧？

**Comment by vincenthychow** [comment]
吃的我真是小白，只能給一點沒作用的意見....

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv (hunk 22 above)
Comment (NoraZheng) [reply subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: 可以用broth

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv (hunk 9 above)
Comment (NoraZheng) [reply subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: five-spice powder

File: subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv (hunk 17 above)
Comment (Yingzhang1122) [reply subtitles/wang-gang/20191012-Niu-Wa-Yu-Tou.sbv]: 酸菜就是经过发酵的菜呀~

**Pull request merged**